### PR TITLE
New package: jack-keyboard-2.7.2

### PR DESCRIPTION
**Pull request by iFoundSilentHouse**
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): system, compiled

- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures:
  - x86_64-musl
  - aarch64-musl (cross)
  - armv7l (cross)

### CODE REVIEW

**Comment by meator** [comment]
You should use the `cmake` build style.

**Comment by iFoundSilentHouse** [comment]
> You should use the `cmake` build style.

Cmake style uses `ninja` build commands. This package needs cmake and then make install.

**Comment by meator** [comment]
CMake supports a variety of build systems. All of them should work equally well. Upstream might recommend `make`, but `ninja` should also work. Void uses CMake with Ninja, so you should prefer that.

**Comment by iFoundSilentHouse** [comment]
@meator thanks a lot!

File: srcpkgs/jack-keyboard/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'jack-keyboard'
+pkgname=jack-keyboard
+version=2.7.2
+revision=1
+build_style=cmake
+configure_args="-DLashEnable=False"
+hostmakedepends="pkg-config"
+makedepends="gtk+-devel jack-devel pango-devel"
```
Comment (classabbyamp) [inline srcpkgs/jack-keyboard/template:8]: we don't want to add any more gtk2 packages

**Pull request closed without merging**